source/installation/install-on-a-local-machine/install-mac.txt

```diff
@@ -30,7 +30,7 @@ Steps
    
       .. include:: /includes/fact-driver-download-locations.rst
 
-   b. Copy the driver files to ``/Library/Application Support/MongoDB/Relational Migrator/Drivers``.
+   b. Copy the driver files to ``~/Library/Application Support/MongoDB/Relational Migrator/Drivers``.
```

Comment:
This is wrong - it should stay with the original path that's absolute to the system root.